watertap/unit_models/pressure_exchanger.py

```diff
@@ -403,31 +514,31 @@ def initialize_build(
         init_log.info_high("Initialize inlets complete")
 
         # check that inlets are feasible
-        if value(self.low_pressure_side.properties_in[0].pressure) > value(
-            self.high_pressure_side.properties_in[0].pressure
+        if value(self.feed_side.properties_in[0].pressure) > value(
+            self.brine_side.properties_in[0].pressure
         ):
             raise ConfigurationError(
                 "Initializing pressure exchanger failed because "
-                "the low pressure side inlet has a higher pressure "
-                "than the high pressure side inlet"
+                "the feed side inlet has a higher pressure "
+                "than the brine side inlet"
             )
         # only needed when there is no mass trnasfer
 
         if (
             abs(
-                value(self.low_pressure_side.properties_in[0].flow_vol)
-                - value(self.high_pressure_side.properties_in[0].flow_vol)
+                value(self.feed_side.properties_in[0].flow_vol)
+                - value(self.brine_side.properties_in[0].flow_vol)
             )
-            / value(self.high_pressure_side.properties_in[0].flow_vol)
+            / value(self.brine_side.properties_in[0].flow_vol)
             > 1e-4
-            and not self.config.has_mass_transfer
+            and not self.config.has_mixing
```

Comment:
I'm not sure if this error message actually helps with initialization. If we need to keep it, maybe we should change self.config.has_mixing to self.config.has_leakage.

Reply:
@luohezhiming please tag this in the issue for tracking any potential updates we may (or may not) want to make in in a future PR. For now, I think we can merge what you have for the March release and refine later, if needed.